src/index.js

```diff
@@ -43,8 +43,9 @@ function ImagekitMediaLibraryWidget() {
       this.callback = arguments[1];
       callbackFunction = this.callback;
     }
-
-    view = this.options.view;
+    if (this.options?.view) {
```

Comment:
Instead of adding an optional chaining for `this.options`, we should ensure that `this.options` is always defined

In the `if` block where the `arguments[0]` is validated, add an else block to set `this.options` to `defaults`